testing.rst

```diff
@@ -484,7 +484,7 @@ script::
 AJAX Requests
 ~~~~~~~~~~~~~
 
-The Client provides a :method:`Symfony\\Component\\BrowserKit\\Client::xmlHttpRequest`
+The Client provides a `xmlHttpRequest`
```

Comment:
The `Client` has been deprecated instead, we should replace it by ``:class:`Symfony\Bundle\FrameworkBundle\KernelBrowser` `` instead. But this should be done everywhere in this file.